README.md

```diff
@@ -42,6 +44,8 @@ Token for the Node infra3.: eyJuYW1lIjoibWFhcy00Lm1hYXMiLCJzZWNyZXQiOiI3MmE51234
 ```
 
 ### Join new nodes to the MAAS cluster
+During installation of the HAProxy charm, you will be asked for a virtual_ip. Pass an empty value to disable it.
```

Comment:
```suggestion

During installation of the HAProxy charm, you will be asked for a `virtual_ip`. Pass an empty value to disable it.
```

Reply:
Maybe we need to investigate a bit to make this question not been asked on joining new nodes.